### PR TITLE
Remove nondeterminism from test

### PR DESCRIPTION
**Pull request by hannahilea**
Default initialization of `Flux.Dense` layer makes use of the `Random.GLOBAL_RNG()`; seed it!

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
 Flux = "0.10.4, 0.11, 0.12"
-Lighthouse = "0.11"
+Lighthouse = "0.11.3"
```
Comment (ararslan) [inline Project.toml:14]: Is this now incompatible with Lighthouse 0.11.2?

File: test/runtests.jl [hunk 2]
```diff
@@ -23,6 +23,7 @@ end
         rng = StableRNG(157)
         classes = ["class_$i" for i in 1:5]
         c, n = 5, 3
+        Random.seed!(0)
```
Comment (ararslan) [inline test/runtests.jl:26]: I wonder if it's possible to set a `StableRNG` as the global RNG...

File: Project.toml (hunk 1 above)
Comment (hannahilea) [reply Project.toml:14]:
```suggestion
Lighthouse = "0.11"
```

File: test/runtests.jl (hunk 2 above)
Comment (hannahilea) [reply test/runtests.jl:26]: yeah, that would be nice...

**Review by ararslan — APPROVED** [review]

**Pull request merged**